### ISSUE TITLE
Orga team edit page: season dropdown

### ISSUE THREAD

**Issue by mkalininait** [rails-girls-summer-of-code/rgsoc-teams#820]
I've just noticed that we have the `Season` dropdown in the "Sources" block of the Orga team edit form:
![screen shot 2017-08-16 at 15 18 55](https://user-images.githubusercontent.com/1745735/29365242-91a0ce8e-8296-11e7-9d6e-81325ce4444f.png)

Does anyone know where it comes from? It looks misplaced in the "Sources" block.

**Comment by klappradla** [comment]
It has been there since _"forever"_ (as in: 2 years 😂 ), but you're right @mkalininait it just does not have its own headline, that's why it looks slightly misplaced.

It could probably go up somewhere around the name field. The form looks a bit as if it was directly moved over from the "normal" namespace, it still has some obsolete checks for whether or not the user is admin and the wording is targeting students. Maybe one should rephrase some of this 😉 

Do you want me to "fix" it?

**Comment by mkalininait** [comment]
@klappradla Ugh. Yeah, the form is chaotic. It would be great to put it in a order.

For both non-orga and orga views: I don't understand why we show all these fields in the "Members" section:

![screen shot 2017-08-19 at 15 15 47](https://user-images.githubusercontent.com/1745735/29486915-735158d8-84f1-11e7-9a01-7e1ec71116dc.png)

In my opinion "Introduction", "Twitter handle", "Github handle" and "Sources" belong to a separate section e.g. "About the team". We can show it for accepted teams only. The field project should be displayed in the orga view only.

If we talk about the orga view, there are a few fields to manage team's status:

![screen shot 2017-08-19 at 15 23 17](https://user-images.githubusercontent.com/1745735/29486977-7123c1f8-84f2-11e7-8b38-f37fa142035d.png)

I think that "Team type", "Season", "Starts on", "Finishes on", "Project" (see https://github.com/rails-girls-summer-of-code/rgsoc-teams/issues/818) should belong to a separate section, e.g. "Team status" or something like this, which is available for orgas only (obviously).  And I don't know what the field "Blog post info" does and where it should go.

This is my vision, I'd love to hear other opinions, too.

**Comment by mkalininait** [comment]
@klappradla I'm sorry, my previous comment goes soooo much beyond the original issue title :D

**Comment by klappradla** [comment]
No worries @mkalininait 😉 

What you say about the structure pretty much all makes sense. But another question upfront: why would admins *either* use the normal form *or* the orga form? In my opinion they should not be able to use the normal one and just have the orga-form as their place to edit teams.

I'm make this an `enhancement`, since I think the "question" part was answered, right?
I'm a bit too busy right now to actually do it, but I'd just assign it to me and then take care of it once I have more time. What do you think? It's not a "bug", so I wouldn't prioritize it super hight...

**Comment by mkalininait** [comment]
@klappradla Thanks for the comment!

You are right that the orga should use only the orga view. When I was talking about the normal view, I was thinking about the students. When our students get accepted to the program and want to edit their team, they see this chaotic form, too: not the 'status' fields, but the ones which should ideally fall in the 'about the team' section. Thus I think we should improve both views: normal for the students and orga for the orgas.

What do you think?

And you are right that it's not a high priority task, not at all.

**Comment by klappradla** [comment]
Sure, I'd also straight look at both forms.

Was just curious about the use cases. If the orga form is "just" one that orga users shel out to if the "normal" form is not capable of everything the want to do or if it's the go to point of they want to edit a team (as said elsewhere, I think the latter sounds more reasonalbe to me)

**Comment by mkalininait** [comment]
@klappradla I agree, the latter sounds better (more consistent) in the currently implemented system.
